template/test.jinja

```diff
@@ -2,3 +2,8 @@
 
 RUN cd /test && \
     python3 serial-cpu.py
+
+RUN git clone --recursive --branch {{ COXETER_VERSION }} --depth 1 https://github.com/glotzerlab/coxeter \
```

Comment:
This one is stubborn. Now it fails with "fatal: could not create work tree dir 'coxeter': Permission denied"
https://dev.azure.com/glotzerlab/software/_build/results?buildId=6556&view=logs&j=79c7dda6-ab1b-5e73-fbce-2303bc4d7002&t=8a96bb7e-076a-5942-818e-39e921248da2&l=5422

This is because the tests are run after a non-privleged user is created one step earlier in `finalize.jinja`. We need to cd into the home directory of that user before we can clone coxeter. I *think* this will work:
```suggestion
RUN cd \
    && git clone --recursive --branch {{ COXETER_VERSION }} --depth 1 https://github.com/glotzerlab/coxeter \
```
At least, it should get you to the next failure :)